pyproject.toml

```diff
@@ -83,17 +83,65 @@ addopts = [
 ]
 xfail_strict = true
 
-[tool.black]
+[tool.ruff]
+# Exclude a variety of commonly ignored directories.
+exclude = [
+    ".bzr",
+    ".direnv",
+    ".eggs",
+    ".git",
+    ".git-rewrite",
+    ".hg",
+    ".ipynb",
```

Comment:
? why would you need to exclude notebooks? also, if you did exclude them, then why are there changes to them included in this PR?